src/main/kotlin/no/nav/syfo/aktivitetskrav/cronjob/PublishExpiredVarslerCronJob.kt

```diff
@@ -0,0 +1,38 @@
+package no.nav.syfo.aktivitetskrav.cronjob
+
+import net.logstash.logback.argument.StructuredArguments
+import no.nav.syfo.aktivitetskrav.AktivitetskravService
+import no.nav.syfo.application.cronjob.Cronjob
+import no.nav.syfo.application.cronjob.CronjobResult
+import org.slf4j.LoggerFactory
+
+class PublishExpiredVarslerCronJob(
+    private val aktivtetskravService: AktivitetskravService
+) : Cronjob {
+    // What should we set here?
+    override val initialDelayMinutes: Long = 2
+    override val intervalDelayMinutes: Long = 10
```

Comment:
Hva er fornuftig interval her? 🤔

Reply:
Tenkte litt på det: Siden dette er noe som går på dags-nivå, så kunne man i teorien kjørt jobben én gang om dagen 🤷🏼‍♂️ Tror ikke det er noe stress å kjøre den hvert 10. minutt, men tror heller ikke det er nødvendig.

Hvis man velger å øke denne, så vil jo kanskje ha andre verdier i `naiserator-dev.yml`-filen slik at man kan feks la den kjøre hvert 10. minutt i dev (for testing purposes), og så litt sjeldnere i prod.